### ISSUE TITLE
identical form names, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#793]
if the form name contains dashes,

```abap
FORM foo-bar.
ENDFORM.

FORM foo-moo.
ENDFORM.
```
